LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Sprig
+MIT License
```

Comment:
Thanks for catching this. The license is not supposed to have this line at all. See https://opensource.org/licenses/MIT. Can you change this to remove it?

Reply:
I am not sure thats correct. if you let github auto-generate a license file for you based on MIT template, it does add the header on top.